airontools/constructors/layers.py

```diff
@@ -69,6 +69,9 @@ def layer_constructor(
             images and 4 for 3D images).
             advanced_reg (bool): Whether to automatically set advanced regularization. Useful to quickly make use of all
             the regularization properties.
+            is_activated_custom (bool): Whether to use a custom Dense layer with keras Functional API or not
+            custom_model_obj (object): Custom Dense Layer object constructed with keras Functional API, activated only
```

Comment:
```suggestion
            custom_layer (Layer): Custom Layer class constructed with keras Functional API
```
It doesn't have to be a dense layer right? as long as it's compatible to the shapes I think it should be fine, then it gives more flexibility. What do you think?

Reply:
yes, that's true.